ttml_test.go

```diff
@@ -68,3 +68,20 @@ func TestTTMLBreakLines(t *testing.T) {
 
 	assert.Equal(t, strings.TrimSpace(string(c)), strings.TrimSpace(w.String()))
 }
+
+func TestTTMLWithOptionsNoIndent(t *testing.T) {
+	// Open
+	s, err := astisub.OpenFile("./testdata/example-in-no-indent.ttml")
```

Comment:
Why add this new `in` file? Can't you use `./testdata/example-in.ttml` directly?

Reply:
I understand but I'd rather use `./testdata/example-in.ttml` here 👍

Reply:
OK, will fix it soon.